### DISCUSSION TITLE
Adding Move Line Shortcuts to the Editor

### DISCUSSION THREAD

**Discussion by J3ekir** [uBlockOrigin/uBlock-issues#2872] [Ideas]
I think it would be a nice feature. I tried a [pull request](https://github.com/gorhill/uBlock/commit/ef08f1f8056001a83e01cd3b436a120c6b943257), but failed.

**Comment by Yuki2718** [comment]
PR is restricted for those who have contribution history.

**Comment by kaya51** [comment]
A nice feature. I sort them too, not too often though.

I  don't think Raymond will add this. We'll get it when he switches to CodeMirror 6.
